opentelemetry-declarative-config/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name = "opentelemetry-declarative-config"
+version = "0.29.1"
+description = "Declarative configuration for OpenTelemetry SDK"
+homepage = "https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-declarative-config"
+repository = "https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-declarative-config"
+readme = "README.md"
+categories = [
+    "development-tools::debugging",
+    "development-tools::profiling",
+]
+keywords = ["opentelemetry", "declarative", "metrics", "configuration"]
+license = "Apache-2.0"
+edition = "2021"
+rust-version = "1.75.0"
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
+
+[dependencies]
+opentelemetry = { version = "0.31.0" }
+opentelemetry_sdk = { version = "0.31.0", features = ["experimental_metrics_custom_reader"] }
+opentelemetry-stdout = { version = "0.31.0" }
+opentelemetry-otlp = { version = "0.31.0" }
+opentelemetry-http = { workspace = true, optional = true, default-features = false }
+serde = { workspace = true, features = ["derive"] }
+reqwest = { workspace = true, optional = true }
```

Comment:
I don't think `opentelemetry-http` and `reqwest` need to be added as direct dependencies. You already have `opentelemetry-otlp` dependency which should transitively pull these two dependencies.

Once you remove these two, you can also remove the `cargo-machete` section as well.